### ISSUE TITLE
weight quession

### ISSUE THREAD

**Issue by 1colorworker** [superhighlevel/ChromaGan_Pytorch_Remake#11]
Can the project code train the effect in the article？
Can you share your trained weights?
my e-mail :2422986674@qq.com

**Comment by superhighlevel** [comment]
I do not guarantee the code will perform the same as the article. But the results are pretty okay.
I can't share weights because we train on private dataset

**Comment by 1colorworker** [comment]
Thanks for your reply!
Also, I'd like to ask how many epochs you trained?
How the loss function is trending.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "superhighlevel/ChromaGan_Pytorch_Remake"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年8月23日(星期二) 下午2:51
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [superhighlevel/ChromaGan_Pytorch_Remake] weight quession (Issue #11)





 
I do not guarantee the code will perform the same as the article. But the results are pretty okay.
 I can't share weights because we train on private dataset

**Comment by superhighlevel** [comment]
About 5 epoch with million pictures I think.
Since it's GAN-base so the loss has quite large fluctuations. Also I don't know why but the Gen loss is low and negative. I think it's the best to evaluation by hand.

**Comment by 1colorworker** [comment]
I have trained 150 epochs on the COCO dataset, but the PNSR is only 20.5, so I would like to ask how effective your training is
